kolibri/core/analytics/management/commands/profile.py

```diff
@@ -0,0 +1,82 @@
+import logging
+import time
+from datetime import timedelta
+
+import psutil
```

Comment:
Yes, but not yet. My plans include trying to do it in pure python if possible. psutil requires C compiled libraries and we are running away from them in the installer. So far, I haven't found any alternative, but once we have a clear decission on the metrics/info to retrieve I will take a last look to the windows world to try to avoid psutil. If there are not alternatives, we'll have to add it to requirements.txt and modify the installers.

Reply:
That's fair - I think what you are measuring should have a chance of finding a platform-independent & pure-python way of working.. preferably without further dependencies of course :)